### ISSUE TITLE
Overlooks All PnP Dependencies

### ISSUE THREAD

**Issue by Kurt-von-Laven** [davglass/license-checker#250]
When you run license-checker in a project that uses [Plug'n'Play dependencies](https://yarnpkg.com/features/pnp), the default in Yarn 2, rather than a node_modules directory, it only sees the package.json of the project itself and doesn't find any of the licenses of any of the dependencies.

**Comment by ferm10n** [comment]
You might want to check out this plugin for yarn https://github.com/tophat/yarn-plugin-licenses

**Comment by Kurt-von-Laven** [comment]
Yes, I am a contributor to that plugin. It doesn't work for us on Yarn >3.0.0 unfortunately on account of tophat/yarn-plugin-licenses#21. I recommend Pivotal's LicenseFinder to others who share this problem since support for Yarn v2+ was recently merged to address pivotal/LicenseFinder#912.
